web/modules/custom/paragraph_align/src/Plugin/paragraphs/Behavior/imagealign.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Drupal\paragraph_align\Plugin\paragraphs\Behavior;
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Annotation\ParagraphsBehavior;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\Entity\ParagraphsType;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+
+/**
+ * @ParagraphsBehavior(
+ *   id = "dummy_image_and_text",
```

Comment:
Wrong identifier of the plugin. 
I think should be 'image_text_align' or something like that.

Reply:
isn't solved